tests/integration/customers.test.js

```diff
@@ -1155,9 +1156,81 @@ describe('CUSTOMER SUBSCRIPTIONS ROUTES', () => {
 
       expect(result.statusCode).toBe(200);
       expect(result.result.data.customer.subscriptions).toBeDefined();
-      expect(result.result.data.customer.subscriptions[1].service._id).toEqual(payload.service);
-      expect(result.result.data.customer.subscriptions[1].versions[0].unitTTCRate)
-        .toEqual(payload.versions[0].unitTTCRate);
+      expect(result.result.data.customer.subscriptions.some(s =>
+        UtilsHelper.areObjectIdsEquals(s.service._id, payload.service))).toBeTruthy();
+      expect(result.result.data.customer.subscriptions.some(s =>
+        s.versions[0].unitTTCRate === payload.versions[0].unitTTCRate)).toBeTruthy();
+    });
+
+    it('should add fixed subscription to customer', async () => {
+      const customer = customersList[2];
+      const payload = {
+        service: service5,
+        versions: [{
+          unitTTCRate: 12,
+          weeklyCount: 12,
+          evenings: 2,
+          sundays: 1,
+        }],
+      };
+
+      const result = await app.inject({
+        method: 'POST',
+        url: `/customers/${customer._id}/subscriptions`,
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload,
+      });
+
+      expect(result.statusCode).toBe(200);
+      expect(result.result.data.customer.subscriptions).toBeDefined();
+      expect(result.result.data.customer.subscriptions.some(s =>
+        UtilsHelper.areObjectIdsEquals(s.service._id, payload.service))).toBeTruthy();
+      expect(result.result.data.customer.subscriptions.some(s =>
+        s.versions[0].unitTTCRate === payload.versions[0].unitTTCRate)).toBeTruthy();
+    });
+
+    it('should return a 422 if try to create fixed service without weeklyCount', async () => {
```

Comment:
Tu pourrais spécifier `should return a 422 if try to create fixed service without weeklyCount and with weeklyHours`

Reply:
Ok ! Est-ce que tu veux que je cree un deuxieme test pour le cas ou il manque uniquement weeklyCount ?

Reply:
Ok ! Est-ce que tu pense qu'il faut creer un deuxieme test pour le cas ou il manque uniquement weeklyCount ?

Reply:
Je me dis qu'il vaut mieux faire deux tests car une souscription forfaitaire sans weeklyCount n'a pas trop de sens --> la condition sur weklyCount est donc importante. D'autant plus qu'on a deux conditions dans le prehandler donc j'aurais tendance a faire le test sur les deux.

Reply:
J'ai change un peu les tests parce que si j'ai ni weeklyCount ni weeklyHours je suis bloquée par la route et pas par le prehandler